packages/flutter_tools/lib/src/devfs.dart

```diff
@@ -267,19 +268,20 @@ class _DevFSHttpWriter {
       HttpClientRequest request = await _client.putUrl(httpAddress);
       request.headers.removeAll(HttpHeaders.ACCEPT_ENCODING);
       request.headers.add('dev_fs_name', fsName);
+      // TODO(goderbauer): transfer real Uri (instead of file path) when remote end supports it
```

Comment:
What is the plan here with respect to changes on the VM side? Are you intending to make these changes?